07-React/react-boilerplate-master/components/movietable.js

```diff
@@ -0,0 +1,51 @@
+var React = require ('react');
```

Comment:
**Minor:** spaces in file names should be represented with *-* (*movie-table.js*).